code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -64,8 +64,8 @@
 		exposed_mob.domutcheck()
 
 /datum/reagent/toxin/mutagen/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
-	affected_mob.adjustToxLoss(0.5 * seconds_per_tick * REM, required_biotype = affected_biotype)
-	return ..()
+	. = affected_mob.adjustToxLoss(0.5 * seconds_per_tick * REM, required_biotype = affected_biotype)
+	return . || ..()
```

Comment:
```suggestion
	. = affected_mob.adjustToxLoss(0.5 * seconds_per_tick * REM, required_biotype = affected_biotype)
	return ..() || . 
```
`. || ..()` won't run `..()` if `.` is `TRUE` already, which we presumably still want to do or the mutagen won't deplete.

Reply:
Alternately
```
	. = ..()
	return affected_mob.adjustToxLoss(0.5 * seconds_per_tick * REM, required_biotype = affected_biotype)
```
Looks like it'll do the same thing more succinctly.